packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/braintreeSummary.gql.js

```diff
@@ -51,6 +51,7 @@ export const GET_BRAINTREE_SUMMARY_DATA = gql`
         }
     }
 `;
+/* eslint-enable graphql/template-strings */
```

Comment:
I don't think this is necessary anymore after #3198. Check that PR for how to include local schema modifications. It _should_ carry over to extensions without too much work.